tests/commands/release/export.spec.ts

```diff
@@ -0,0 +1,109 @@
+import * as stream from 'node:stream';
+import { cleanOutput, runCommand } from '../../helpers';
+import { BalenaAPIMock } from '../../nock/balena-api-mock';
+import { expect } from 'chai';
+import * as mock from 'mock-require';
+import * as sinon from 'sinon';
+import { promises as fs } from 'fs';
+
+// "itSS" means "it() Skip Standalone"
+const itSS = process.env.BALENA_CLI_TEST_TYPE === 'standalone' ? it.skip : it;
+
+describe('balena release export', function () {
+	const appCommit = 'badc0ffe';
+	const appSlug = 'testOrg/testApp';
+	const appVersion = '1.2.3+rev1';
+	const outputPath = '/tmp/release.tar';
```

Comment:
@cmfcruz I suppose these tests are breaking on windows because this is not a valid path on windows (escaping using `/` vs `\\`). You can use nodejs `path` library with a join function to solve this, e.g.

```suggestion
import * as path from 'node:path';
...
const outputPath = path.join('tmp', 'release.tar');
```

Reply:
Code updated.  Thanks for the clue, @otaviojacobi .